buildSrc/src/main/groovy/io/radar/mvnpublish/DefaultRadarPublication.groovy

```diff
@@ -0,0 +1,95 @@
+package io.radar.mvnpublish
+
+import org.gradle.api.Action
+import org.gradle.api.Project
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+
+class DefaultRadarPublication implements RadarPublication {
```

Comment:
I kind of wonder if we even need this whole class hierarchy, given there's only one child.

Reply:
You mean we can remove the interface and just name this class RadarPublication?

Reply:
I think that's what he means, there's nothing else implementing this right?

Reply:
Removed/cleaned up